components/CopyCodeBlock/copyCodeBlock.module.css

```diff
@@ -0,0 +1,51 @@
+/* Styles for the button before being clicked */
+
+.codeCopyBtn {
+	color: white;
+	position: absolute;
+	right: 5px;
+	top: 5px;
+	font-size: 1rem;
+	cursor: pointer;
+	transition: all 0.3s ease-in-out;
+}
+
+.codeCopyBtn:hover {
+	transform: scale(1.1);
+	opacity: 0.9;
+}
+
+.codeCopyBtn:hover::after {
+	content: "copy";
+	display: flex;
+	align-items: center;
+	font-size: 0.5rem;
+	opacity: 0.9;
+}
+
+.codeCopyBtn:focus::after {
+	content: "copy";
+	display: flex;
+	align-items: center;
+	font-size: 0.5rem;
+	opacity: 0.9;
+}
+
+/* Styles for button after it has been clicked */
+.codeCopied {
```

Comment:
This is a repetition of the codeCopyBtn class so it not needed. They can be combined into one class, especially if the ::after is removed and added as text to the button. Then that can be condidtionally rendered by the state inside the component